src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -48,13 +48,13 @@ public void storeMessage(Message message) {
     messageEntity.setProperty("category", message.getCategory());
     messageEntity.setProperty("latitude", message.getLat());
     messageEntity.setProperty("longitude", message.getLng());
+    messageEntity.setProperty("image", message.getImageUrl());
     datastore.put(messageEntity);
   }
 
   public void deleteMessageWithID(String messageID) {
-    Query query =
-        new Query("Message")
-            .setFilter(new Query.FilterPredicate("ID", FilterOperator.EQUAL, messageID));
+    Query query = new Query("Message")
```

Comment:
Please run "mvn com.coveo:fmt-maven-plugin:format" to remove these unrelated formatting changes.

Reply:
Done